index.html

```diff
@@ -8,6 +8,12 @@
       <div id="test-area">
           <div id="textbox"></div>
           <div id="input-area">
+            <div class="settime">
+            <button id="15"> 15 </button>
+            <button id="30"> 30 </button>
+            <button id="45"> 45 </button>
+            <button id="60"> 60 </button>
```

Comment:
Instead of using `id`, use `value` field since that is more fitting

Reply:
Also the div that is currently selected should have the text white and the other unselected ones grey to show which one is currently selected. Better than buttons.

See https://monkeytype.com/ implementation of how the time is selected.